app/views/shared/_wysiwyg.slim

```diff
@@ -1,4 +1,5 @@
-#summernote
+.wysiwyg id=id
+= f.hidden_field :content, id: "#{id}_content"
 
 javascript:
-  $.publish("wysiwyg:setup");
+  $.publish("wysiwyg:setup", "#{id}");
```

Comment:
`id` is a string - `slim` will display it without quotation marks, which'll result in a JS `Uncaught ReferenceError`.